src/_cffi_src/openssl/x509_vfy.py

```diff
@@ -266,36 +266,9 @@
 static const long X509_V_FLAG_TRUSTED_FIRST = 0;
 #endif
 
-/* OpenSSL 1.0.0+ verification error codes */
-#if OPENSSL_VERSION_NUMBER >= 0x10000000L
 static const long Cryptography_HAS_100_VERIFICATION_ERROR_CODES = 1;
```

Comment:
I think we can probably remove this as it's unlikely that anybody used this as a feature check.